src/Microsoft.Health.Fhir.Core/Features/Search/SearchService.cs

```diff
@@ -105,9 +146,27 @@ public async Task<Bundle> SearchCompartmentAsync(string compartmentType, string
             {
                 queryParameters.Add(Tuple.Create(SearchParameterNames.LastUpdated, at.ToString()));
             }
-            else if (since != null)
+            else
             {
-                queryParameters.Add(Tuple.Create(SearchParameterNames.LastUpdated, $"ge{since}"));
+                if (since != null)
+                {
+                    queryParameters.Add(Tuple.Create(SearchParameterNames.LastUpdated, $"ge{since}"));
+
+                    // when since is specified without _before, we will assume a value for before that is
+                    // Now - <buffer>  the buffer value allows us to have confidence that ongoing
+                    // edits are captured should the _before value be used in a future _history query
+
+                    if (before == null)
+                    {
+                        addedBefore = Clock.UtcNow.AddSeconds(historyCurrentTimeBuffer);
+                        before = new PartialDateTime(addedBefore.Value);
```

Comment:
nit: more efficient if this just simply add the query parameter directly. Creating a new PartialDateTime requires parsing of an already fully populated DateTimeOffset into individual component then stich it together again using StringBuilder.